src/commands/deployWorkspaceProject/settings/convertSettings/DetectV1SettingsStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra, AzureWizardPromptStep, DialogResponses, nonNullProp } from "@microsoft/vscode-azext-utils";
+import type * as vscode from 'vscode';
+import { localize } from "../../../../utils/localize";
+import { settingUtils } from "../../../../utils/settingUtils";
+import { type ConvertSettingsContext } from "./ConvertSettingsContext";
+
+export class DetectV1SettingsStep extends AzureWizardPromptStep<ConvertSettingsContext> {
+    public async prompt(context: ConvertSettingsContext): Promise<void> {
+        const settingsPath: string = settingUtils.getDefaultRootWorkspaceSettingsPath(nonNullProp(context, 'rootFolder'));
+        const settingsContents: string = await AzExtFsExtra.readFile(settingsPath);
```

Comment:
Do we need to add a `pathExists` check here?  I can't remember but I thought that an error might throw if the path doesn't exist when we go to read the file